uxarray/grid.py

```diff
@@ -206,6 +209,12 @@ def __from_ds__(self, dataset):
             self.ds, self.ds_var_names = _read_ugrid(dataset, self.ds_var_names)
         elif self.mesh_type == "shp":
             self.ds = _read_shpfile(dataset)
+        elif self.mesh_type == "mpas":
+            # select whether to use the dual mesh
+            if self.use_dual is not None:
```

Comment:
Why are you using `is not None` here? Wouldn't this theoretically produce unexpected results if `self.use_dual` is set, but set to False?

Reply:
Since `use-dual` is part of the `**kwargs` of our `Grid` class, it's set to `None` if the user doesn't pass it through. See the usage example in the description for how it's supposed to be used.

Reply:
Ah, yeah I see that now!